state-transition/core/validation_deposits_test.go

```diff
@@ -263,8 +261,7 @@ func TestLocalDepositsExceedBlockDeposits(t *testing.T) {
 }
 
 func TestLocalDepositsExceedBlockDepositsBadRoot(t *testing.T) {
-	csData := spec.BaseSpec()
-	csData.DepositEth1ChainID = spec.BoonetEth1ChainID
+	csData := defaultSpecData()
```

Comment:
q: should we use Testnet or devnet values instead of yet another test config?

Reply:
most UTs are using mainnet spec. This is required if modifications are necessary to test